Code/GraphMol/MolAlign/AlignMolecules.cpp

```diff
@@ -30,7 +30,10 @@ double getAlignmentTransform(const ROMol &prbMol, const ROMol &refMol,
   if (atomMap == 0) {
     // we have to figure out the mapping between the two molecule
     MatchVectType match;
-    if (SubstructMatch(refMol, prbMol, match)) {
+    const bool recursionPossible = true;
```

Comment:
This set of changes is backwards incompatible.
If you want control over these matching options, feel free to add optional arguments to the function.